test/drbg_cavs_test.c

```diff
@@ -254,6 +254,11 @@ static int test_cavs_kats(const struct drbg_kat *test[], int i)
     const struct drbg_kat *td = test[i];
     int rv = 0;
 
+#ifdef FIPS_MODE
+    /* FIPS mode doesn't support CTR DRBG without a derivation function */
+    if ((td->flags & USE_DF) == 0)
+        return 1;
```

Comment:
I'm not sure this is the correct return value. Do we have something that indicates skipped instead of pass?

Reply:
Not that I can find.

Reply:
It is a good suggestion to have a skip outcome.

Reply:
Skip is awaiting approval of #8695